app/serializers/person_serializer.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
-class PersonSerializer < ActiveModel::Serializer
+class PersonSerializer
+  include JSONAPI::Serializer
+
+  set_type :people
```

Comment:
Given it's called the JSONAPI gem, I'm genuinely confused about why it doesn't infer the plural form for type 🤷 . Or has Dom been saying we always need plural forms for json:api when we don't 😆

Reply:
It is weird, but without this line in each serializer it does indeed infer the singular form from the name, i.e. this would populate a type of `person`. c'est la vie.

Reply:
Yeah. Crazy, but fast, JSONAPI gem